net/fail2ban/Makefile

```
@@ -20,6 +20,8 @@ PKG_LICENSE_FILES:=COPYING
 include $(INCLUDE_DIR)/package.mk
 include ../../lang/python/python3-package.mk
 
+HOST_BUILD_DEPENDS:=python3/host 2to3/host
```

```suggestion
HOST_BUILD_DEPENDS:=2to3/host
```

Since `2to3/host` already depends on `python3/host`.

---

okay, agreed, thanks, modified

---

But still non testable with actual master...
Will have to switch to my local 2to3 branch, because of still missing 2to3 host package.
https://github.com/openwrt/packages/pull/17429 still pending !

---

It looks like to work, my test gets successful.
The package was built.
Working also on TestBuild...
But if the 2to3 package is missing, it might not work !
I do not understand why it is built without the 2to3 package...
I get some same issue/success with the initial package, without get the point to understand...
May be a CACHE problem or any compilation side effect !?
The issue https://github.com/openwrt/packages/issues/17311 came because of the missing 2to3 !!!
Why it might or may not work, is non-understandable for now.